src/components/Table/CustomTable.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { Table, Pagination, Select } from 'antd';
+import './CustomTable.scss';
+
+const CustomTable = ({ data, columns }) => {
+  const [pageSize, setPageSize] = useState(10);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const handlePageSizeChange = (value) => {
+    setCurrentPage(1);
+    setPageSize(parseInt(value));
+  };
+
+  const handlePageChange = (page) => {
+    setCurrentPage(page);
+  };
+
+  const totalItems = data.length;
+
+  const startIndex = (currentPage - 1) * pageSize;
+  const endIndex = startIndex + pageSize;
+  const pageData = data.slice(startIndex, endIndex);
+
+  const paginationConfig = {
+    pageSize: pageSize,
+    total: totalItems,
+    current: currentPage,
+    onShowSizeChange: handlePageSizeChange,
+    onChange: handlePageChange,
+    showTotal: (total, range) => `${range[0]}-${range[1]} of ${total} items`,
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < Math.ceil(totalItems / pageSize)) {
+      setCurrentPage(currentPage + 1);
+    }
+  };
+
+  const handleLastPage = () => {
+    setCurrentPage(Math.ceil(totalItems / pageSize));
+  };
+
+  return (
+    <div className="table-container">
+      <Table dataSource={pageData} columns={columns} className="table" pagination={false} />
+
+      <div className="pagination-container">
```

Comment:
You can move this whole part in separate component in Tables/components folder
You can call them something like noOfRows or something like that